lib/api/objectDeleteTagging.js

```diff
@@ -71,10 +71,12 @@ function objectDeleteTagging(authInfo, request, log, callback) {
             objectMD.tags = {};
             const params = objectMD.versionId ? { versionId:
               objectMD.versionId } : {};
-            const replicationInfo = getReplicationInfo(objectKey, bucket, true);
+            const replicationInfo = getReplicationInfo(objectKey, bucket, true,
+                0, 'DELETE_TAGGING');
```

Comment:
Please create a constant after the requires, this way we don't have to hunt for these constants to maintain them.
```js
const REPLICATION_ACTION = 'DELETE_TAGGING';
```

Reply:
Updated.